app/controllers/application_controller.rb

```diff
@@ -108,4 +109,8 @@ def current_organisation
     @current_organisation ||= Organisation.find_by(id: session[:publisher_organisation_id])
   end
   helper_method :current_organisation
+
+  def user_for_paper_trail
+    current_publisher
```

Comment:
Possibly `current_publisher || current_support_user` to future-proof this? Given that soon they will be able to edit stuff too!